pkgs/development/python-modules/pygls/default.nix

```diff
@@ -3,7 +3,7 @@
   stdenv,
   buildPythonPackage,
   fetchFromGitHub,
-  lsprotocol,
+  lsprotocol2023,
```

Comment:
While we are here, could we add a version regex pinning pygls to 1.x versions? Otherwise the bot will keep trying to update to alphas.